src/react/index.js

```diff
@@ -16,7 +16,7 @@ export {
 
 const push = (obj, items) => Array.prototype.push.apply(obj, items)
 
-export default function (tag, objs, vars = [], content) {
+export default function (tag, cls, objs, vars = [], content) {
```

Comment:
I am in 100% agreement on moving it all into content. I'm going to merge this in as is but I think we should focus on that next.